_includes/about.html

```diff
@@ -4,19 +4,16 @@
         <div class="col8 clearfix space-bottom2 contain">
           <div class="col10 margin2r space-bottom2">
             <h1 class="space-bottom2">FOSS4G SotM Oceania 2020</h1>
-      <!--	    <p> <a href="https://www.picgisrs.org/conference-venue/" rel="noopener noreferrer" target="_blank">Supporting the Pacific GIS/RS Conference in Vanuatu 2020</a> </p> -->
             <p> <h2>About the Conference:</h2>
             <p>The FOSS4G SotM Oceania Conference was planned to be an in-person international conference in Suva, Fiji. Unfortunately, we aren't able to travel to Fiji, so we're having a hybrid event.</p>
             <p>This year's event is a hybrid online/in-person event, with hubs being organised in cities across the region, from Perth in the west to Rarotonga in the east, and many places in between.</p>
             <p>The conference has two components:</p>
-            * A broadcast of keynote presentations from across the region, bringing the brightest and most thought-provoking ideas to your screen.
```

Comment:
Is this supposed to be an unordered list?

And makbe we change it to just be a normal paragraph, and add a prominent button to register instead. Thoughts? I can work on the HTML if you like.

Reply:
Thanks Alex, my html is pretty lame.